client/constants/Constants.jsx

```diff
@@ -0,0 +1,38 @@
+export const AUTH = {
```

Comment:
I believe this file doesn't have anything to do with JSX, so I should be a `.js` file